src/adapter/homeAdapter/index.js

```diff
@@ -86,6 +113,34 @@ export function handleEdgeClick (edgeUrl, edgeName){
 
 }//handleEdgeClick
 
+/**
+ * this returns the default dot language for dedup application
+ */
+export function defualt_dedup_input() {
```

Comment:
Do not hardware the testing input into the code. Put different tests into separate files. Use separate systems for testing. Testing input should not be part of the code at all.